README.md

```diff
@@ -39,6 +39,55 @@ library = foreman-ansible-modules/modules
 
 Now your playbooks and roles should have access to the modules contained in the repository for use, testing or development of new modules.
 
+## How to debug this repository
+
+Clone both ansible and this project
+
+```sh
+git clone https://github.com/ansible/ansible.git
+git clone https://github.com/theforeman/foreman-ansible-modules.git
```

Comment:
Do I re-clone this? or clone it inside the repository? If I am debugging, its fair to assume I already have this repository cloned?